test/extensions/quic_listeners/quiche/integration/quic_http_integration_test.cc

```diff
@@ -391,9 +393,13 @@ TEST_P(QuicHttpIntegrationTest, TestDelayedConnectionTeardownTimeoutTrigger) {
             1);
 }
 
-TEST_P(QuicHttpIntegrationTest, MultipleQuicConnectionsWithBPF) { testMultipleQuicConnections(); }
+// Ensure multiple quic connections work, regardless of platform BPF support
+TEST_P(QuicHttpIntegrationTest, MultipleQuicConnectionsDefaultMode) {
```

Comment:
@ggreenway renamed, let me know what you think (removed BPF from the name)

Reply:
Yes, that looks better, thanks